src/actions/bucket.js

```diff
@@ -65,8 +65,8 @@ export function listBucketGroupsSuccess(groups: Object[]): Action {
   return {type: BUCKET_GROUPS_SUCCESS, groups};
 }
 
-export function listBucketHistory(bid: string, since: Number): Action {
-  return {type: BUCKET_HISTORY_REQUEST, bid, since};
+export function listBucketHistory(bid: string, params: Object = {}): Action {
```

Comment:
Not sure about the `params` name, which one may confuse with the route params; how about `filters`?

Also, this would probably deserve a dedicated type, eg. `HistoryFilters` :)